app/Factories/AlbumFactory.php

```diff
@@ -153,6 +153,7 @@ public function findBaseAlbumsOrFail(array $albumIDs, bool $withRelations = true
 			$albumQuery->with(['photos', 'children', 'photos.size_variants']);
 		}
 
+		/** @var Collection<BaseAlbum> $result */
```

Comment:
As far as I know, for Laravel 9 and Larastan 2, `Collection` needs two type parameters. The type of the key (either `string` or `int`) and the type of the value.

Reply:
fixed [a066720](https://github.com/LycheeOrg/Lychee/pull/1469/commits/a0667203456e97901e8985e4236d633c788a3336)